Hot Doge Token

DiamondHandsCorp opened the pull request:
Thank you!

DiamondHandsCorp commented:
I don't know what happened. Can you fix the issue?

Bal-Gu commented:
Bal Gu [Shadowmod+], [16.01.2022 16:40]
[En réponse à Sean - ÐÄÎʍȭԤԀԊдղḌ]
Correct. The tests are saying:

Please provide a logo.png

Calling it hotdogelogogithub.png is not equal to logo.png 

To this your json file is invalid it is missing some , in the social and in the general fields. Pass it threw an online parser such that it shows you the errors

The rest seems about right ^^

@DiamondHandsCorp

DiamondHandsCorp commented:
HOTDOGE (Hot Doge Token v3) Hopefully this is now correct. Deleted the previous non-working and replaced with this which should be in line with your requirements. Thanks.
